DESCRIPTION

```diff
@@ -45,14 +46,10 @@ Imports:
     survival,
     magrittr,
     ggplot2,
-    patchwork,
     tibble,
     tidyr,
     dplyr,
-    ggpubr,
-    pammtools,
     kableExtra,
-    assertthat,
     gt (>= 0.1.0),
     tidyselect,
     parcats (>= 0.0.1),
```

Comment:
we dont need this one. as.factor and factor should be sufficient from base R

Reply:
Removed